README.md

```diff
@@ -150,28 +137,35 @@ To run the pipeline, enter the following command into the terminal:
 |
 |   create_publication.py             <- Runs the overall pipeline to produce the publication     
 |
++---data_in                           <- Stores imported data that will be used to create the outputs
+|   |       .gitkeep                  <- Empty file that enables the data_in folder to be committed
++---data_out                          <- Outputs and aggregations created by the pipeline are stored here
+|   |       .gitkeep                  <- Empty file that enables the data_in folder to be committed
 +---src                               <- Scripts with functions for use in 'create_publication.py'. Contains project's codebase.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |
-|   +---utils                     <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
+|   +---utils                         <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |       file_paths.py             <- Configures file paths for the package
 |   |       logging_config.py         <- Configures logging
-|   |       data_connections.py       <- Handles data connections i.e. reading/writing dataframes from SQL Server
+|   |       spark.py                  <- For setting up and configuring Apache Spark
 |   | 
 |   +---processing                    <- Scripts with modules containing functions to process data i.e. clean and derive new fields
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |       clean.py                  <- Perform cleaning and wrangling processes 
-|   |       derive_fields.py          <- Create new field definitions, columns, derivations.
+|   |       aggregate_counts.py       <- Creates aggregations and counts based on the imported data
```

Comment:
I wonder if we should have put this within the aggregate counts file...

Reply:
I feel like aggregate_counts is more direct as to what's happening in there. When I think of derive fields, I think more like adding new columns to the existing data, like creating age band from an age or something.

But I guess it's six of one, half a dozen of the other! happy to go with derive_fields if you prefer that.